cli/dune

```diff
@@ -1,4 +1,12 @@
 (library
  (name duniverse_cli)
- (libraries duniverse_lib fmt.cli fmt.tty logs.cli logs.fmt cmdliner lwt.unix
-   dune-build-info ocaml-version))
+ (libraries
+  duniverse_lib
```

Comment:
This formatting change should have triggered a CI failure. Trying to format the repo locally reformats them as before.

Reply:
That's strange. Given dune reformats its own files, this shouldn't have anything to do with ocamlformat. So it must be the dune version. I just tried on `main` (with this merged) and dune 2.9.1 keeps them. Maybe some change in the dune formatting defaults?

Reply:
Something changed but these changes are tied to `lang dune`. So if your editor integration uses `dune format-dune-file` instead of `build @fmt` it might pick the wrong version. And CI for opam-monorepo projects does not have a lint-fmt build at the moment so it's not checked in CI.

Reply:
I talked too fast, there's a lint-fmt build now apparently